app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/SelectShopperActivity.java

```diff
@@ -0,0 +1,85 @@
+package com.iic.shopingo.ui.request_flow.Activities;
+
+import android.os.Bundle;
+import android.support.v7.app.ActionBarActivity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnItemClick;
+import com.iic.shopingo.R;
+import com.iic.shopingo.ui.request_flow.Views.SelectShopperListItemView;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SelectShopperActivity extends ActionBarActivity {
+  @InjectView(R.id.select_shopper_list)
+  ListView mShopperList;
+
+  private SelectShopperAdapter mAdapter;
```

Comment:
Let's ditch the "m" prefix from variable names, it's mostly a C++ convention (even though the AOSP adopted it).
But from looking in open source libraries for android such as retrofit, butterknife, ORMLite, I see that none of them use this "m" prefix convention.

Also it makes it harder to generate getters/setters (i.e. if you have a member called mBlat then it generates getmBlat() method).